src/controllers/shipping-tags.js

```diff
@@ -98,6 +98,7 @@ export default function () {
                     <li>${(to.phone ? formatPhone(to.phone) : '')}</li>
                     <li class="pt-1 fs-18"><mark>${to.zip.replace(/(\d{5})(\d{3})/, '$1-$2')}</mark></li>
                     <li class="pt-1 fs-18"><address>${ecomUtils.lineAddress(to)}</address></li>
+                    ${(to.near_to ? `<li class="pb-1">Referência: ${to.near_to}</li>` : '')}
```

Comment:
```suggestion
                    ${(to.near_to ? `<li class="pt-1">Referência: ${to.near_to}</li>` : '')}
```

Pra quê padding bottom no último item? Acho que seria `pt`, não?

Reply:
Achei ele muito colado do número do pedido e forma de envio, por isso um padding bottom pra descolar um pouco. Se não coloco, fica assim:

![sem-pb](https://user-images.githubusercontent.com/35343551/108890020-cce11700-75eb-11eb-8f6c-b6aa449898e4.png)

Quando coloco:

![com-mb](https://user-images.githubusercontent.com/35343551/108890041-d2d6f800-75eb-11eb-9d6a-e5949a9dbb4d.png)

Reply:
A tag address já tem um margin bom ali